src/parsers/forecast_parser.jl

```diff
@@ -293,3 +293,42 @@ function parse_json(filename,device_names)
     return Devices
 end
 =#
+
+"""
+    remove_all_forecasts(sys::System, T::Type{T})::Vector{T} where T <: Forecast
+
+Remove all forecast objects of concrete type T from a System
+"""
+function remove_all_forecasts!(sys::System, ForecastType::Type{T})where T <: Forecast
```

Comment:
I thought we wanted to remove all forecasts regardless of type.  Is that not true?

Reply:
based on my tests, a system can have forecasts of different types that are inconsistent. Therefore, the need to remove forecasts from the system before adding split forecasts only applies to forecasts of the same type.